BlocQuery/BlocQuery/ViewController.m

```diff
@@ -8,19 +8,18 @@
 
 #import "ViewController.h"
 
-@interface ViewController ()
+@interface ViewController () <PFLogInViewControllerDelegate>
 
 @end
 
 @implementation ViewController
 
-- (void)viewDidLoad {
-    [super viewDidLoad];
-
-    PFObject *testObject = [PFObject objectWithClassName:@"TestObject"];
-    testObject[@"foo"] = @"bar";
-    [testObject saveInBackground];
-
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:YES];
+    
+    PFLogInViewController *logInVC = [[PFLogInViewController alloc] init];
```

Comment:
Oh ok. I assumed these might be in the next checkpoint (aside from the delegate) since the checkpoint did not specify. I can see how these seem obvious to include here.